### PR TITLE
Bird Species Classification and Recognition

### PR DESCRIPTION
**Pull request by harshita9621**
## Related Issue 

- This csv file has a total of 41,218 rows, one row for each image and 3 columns. The file path column is the path to the image file. Use os.path.join(input directory, this path) to get the absolute path to the file. The label column contains the name of the bird species associated with the image. The data set column shows which dataset (train, test, valid) the file originally was located.

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [x] Contributor

Closes: #129 

### Describe the add-ons or changes you've made

- In this project, I will classify images of birds by using transfer learning from a pre-trained network. 

- I am using the MobileNet V2 model developed at Google. I am beginner for computer vision and i am trying to learn more about it. I follow transfer learning and fine-tuning which is tensorflow for the project.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how it has been tested
Describe how have you verified the changes made

## Checklist:

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (Only for Front End and UI/UX Designers)

 Original           | Updated
 :--------------------: |:--------------------:
 Original Screenshot | Updated Screenshot |

### CODE REVIEW

**Comment by Manasi2001** [comment]
Very well done @harshita9621! Good work. Keep it up :)
